### PR TITLE
update policy tests from mt to mmt

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: Tools/Policies/policyset2policytransform2policyresult.xsl [hunk 1]
```diff
@@ -175,7 +175,7 @@
     <xsl:variable name="equationfull">
       <xsl:choose>
         <xsl:when test="@scope='mt'">
```
Comment (ablwr) [inline Tools/Policies/policyset2policytransform2policyresult.xsl]: but scope remains 'mt'?

Comment (dericed) [inline Tools/Policies/policyset2policytransform2policyresult.xsl]: good point, one min

**Comment by dericed** [comment]
PR is updated to use micromediatrace is `scope="mmt"`. It also allows `scope="mt"` but you'd need to know which form of the trace data you have when setting scope. So @tribouille please change the GUI to set scope to `mmt` when writing policies.

**Comment by dericed** [comment]
squashed and ready for re-review. I added a file sample_policy_5... as a test of the micromediatrace-based policy tests.

**Comment by JeromeMartinez** [comment]
sample_policy_1.xml is not updated accordingly.

**Comment by dericed** [comment]
@JeromeMartinez sample_policy_1.xml is updated and pushed.

**Pull request merged**